content/cpi-api-v2-method/create-vm.md

```diff
@@ -0,0 +1,164 @@
+# create_vm
+
+Creates a new VM based on the stemcell. The newly created VM must be powered on and accessible on the provided networks.
+
+Waiting for the VM to finish booting is not required because the Director waits until the Agent on the VM responds back.
+
+If the VM's creation fails, please make sure to properly delete the associated resources.
+
+As of V2 of the CPI API contract, `create_vm` returns an array of the resultant instance ID and the networks associated with the VM.
+
+
+## Arguments
+
+ * `agent_id` [String]: ID selected by the Director for the VM's agent.
+ * `stemcell_cid` [String]: Cloud ID of the stemcell to use as a base image for new VM.
+ * `cloud_properties` [Hash]: Cloud properties hash specified in the deployment manifest under the VM's resource pool.
+ * `networks` [Hash]: Networks hash that specifies which VM networks must be configured.
+ * `disk_cids` [Array of strings] Array of disk cloud IDs for the disks that the created VM will most _likely_ attach. The disk cloud IDs could be used to optimize VM placement so that disks are located nearby.
+ * `environment` [Hash]: Resource pool's env hash specified in the deployment manifest, including initial properties added by the BOSH director as shown below. Additionally, the director will append the following guaranteed values:
```

Comment:
Shouldn't we document somewhere that this argument is expected to be passed to the agent as is via the agent settings?